app/constants/herd.js

```diff
@@ -0,0 +1,4 @@
+export const OTHERS_ON_SBI = {
```

Comment:
is there any value in this being in the shared library? Fair enough if its only going to be used here

Reply:
makes sense